versioned_docs/version-2.0.0/keploy-explained/debugger-guide.md

```diff
@@ -44,11 +47,17 @@ Navigate to `launch.json` to begin crafting JSON objects.
       "asRoot": true,
       "console": "integratedTerminal",
       "program": "main.go",
-      "args": ["test", "-c", "<path_to_executable>"]
+      "args": ["test", "-c", "<path_to_executable>"],
+      "env": {
+        "PATH": "${env:PATH}"
+      }
     }
   ]
 }
 ```
+## How to obtain PATH:
+### In Linux: Open the terminal --> write "echo $PATH"
+### In Windows: Open the cmd or powershell --> write "echo %PATH%"
```

Comment:
instead of `-->` write the standard code snippet can be followed .

Like instead of how to obtain~

`Retrieving the PATH Environment Variable`

something like would look better
<img width="687" alt="image" src="https://github.com/keploy/docs/assets/60891544/bcef7d66-2ea7-4e34-8bd9-1c734d719179">

Reply:
@shivamsouravjha  I reformatted the readme to be like what you said.